### ISSUE TITLE
transforms: ToTensor(), Normalization

### ISSUE THREAD

**Issue by kmittle** [mseitzer/pytorch-fid#112]
The InceptionV3 model in PyTorch is trained on ImageNet. According to the official documentation, it uses the mean and standard deviation of ImageNet for normalization. However, I noticed in the code that the normalization method used is just `ToTensor()`. Wouldn't it be more appropriate to normalize using the mean and standard deviation of ImageNet?

![image](https://github.com/mseitzer/pytorch-fid/assets/60066336/c5ffeb5a-07fd-4f4f-9eca-4af238db93fe)
https://pytorch.org/vision/stable/models/generated/torchvision.models.inception_v3.html

**Comment by kmittle** [comment]
Correction: I found that the way to normalize is to normalize to -1 to 1. However, this does not seem to make as much sense as using ImageNet's mean-variance normalization.

**Comment by mseitzer** [comment]
Note that the weights used by pytorch-fid are not the ones from torchvision, but a conversion from a particular checkpoint of Inception from Tensorflow that was used by the original implementation of FID. This particular model was trained under inputs from [-1, 1], not using ImageNet normalization.
